docs/_includes/nav_list

```diff
@@ -18,20 +18,19 @@
             {% assign menu_label = "文档菜单" %}
         {% endif %}
     {% elsif page.version == "0.5.1" %}
-            {% assign navigation = site.data.navigation["0.5.1_docs"] %}
+        {% assign navigation = site.data.navigation["0.5.1_docs"] %}
```

Comment:
why these changes?

Reply:
Here is a bug, it'll cause build failed

![image](https://user-images.githubusercontent.com/20113411/76991189-22e11f80-6984-11ea-845b-1a697c8bc391.png)

Update

![image](https://user-images.githubusercontent.com/20113411/76991411-83705c80-6984-11ea-8bc9-ab052e1f009b.png)